marimo/_plugins/ui/_impl/from_anywidget.py

```diff
@@ -98,11 +111,56 @@ def on_change(change: T) -> None:
                 "css": css,
             },
             on_change=on_change,
+            functions=(
+                Function(
+                    name="send_to_widget",
+                    arg_cls=SendToWidgetArgs,
+                    function=self.receive_from_frontend,
+                ),
+            ),
         )
 
+    def send_to_frontend(
```

Comment:
This should start with an underscore -- don't want to add to  public API right? Same for receive_from_frontend. Can expose as needed on individual UI elements, like you've done for ui.anywidget